src/couch_jwt_auth.erl

```diff
@@ -58,6 +58,12 @@ decode(Token, Config) ->
       end, List), posix_time(calendar:universal_time()), Config)
   end.
 
+ensure_safe_token(Token, Config) ->
+  case couch_util:get_value(Token, Config) of
```

Comment:
I'd say it could be any number, it's really up to whomever runs the service.
Is this the fn you're talking about? https://github.com/apache/couchdb/blob/1.6.x/src/couchdb/couch_config.erl#L59
How would you test this?:

```erl
ensure_safe_token(Token) ->
 case couch_config:get("jwt_auth_blacklist", Token) of
 ...
```